test/output/3.sh.txt

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+ff()
+{
+  while read FILE; do
+    echo "$FILE"
+
+  done <<END
+	$(list "$@")
```

Comment:
Yes. I plan to do so tomorrow.

On Mon, Jun 20, 2016, 00:18 Kyeong Yoo <notifications@github.com> wrote:

> In test/output/3.sh.txt
> <https://github.com/hermanbergwerf/bashbeautify/pull/3#discussion_r67624899>
> :
>
> > @@ -0,0 +1,27 @@
> > +#!/bin/bash
> > +
> > +ff()
> > +{
> > +  while read FILE; do
> > +    echo "$FILE"
> > +
> > +  done <<END
> > +	$(list "$@")
>
> Do you have a plan to push the latest changes to pypi? If that happens
> soon, that would be great. Cheers, Kyeong
> … <#m_-645747876308917508_>
> On 20/06/16 09:31, Herman Bergwerf wrote: In test/output/3.sh.txt <#3
> (comment)
> <https://github.com/hermanbergwerf/bashbeautify/pull/3#discussion_r67624176>>:
> > @@ -0,0 +1,27 @@ > +#!/bin/bash > + > +ff() > +{ > + while read FILE; do
> > + echo "$FILE" > + > + done <<END > + $(list "$@") Aha, I see. I did not
> know about the here document syntax, sorry :-). — You are receiving this
> because you authored the thread. Reply to this email directly, view it on
> GitHub <
> https://github.com/hermanbergwerf/bashbeautify/pull/3/files/90a3c5228a0ad599630a1ac26fee823357316e85#r67624176>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ATBnfeTEDdXDmlYFuJ5f4QqBLABSHh79ks5qNbU9gaJpZM4I2N9T>.
>
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hermanbergwerf/bashbeautify/pull/3/files/90a3c5228a0ad599630a1ac26fee823357316e85#r67624899>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEQJ1UgzT8tpw1oFsHpLmjbf8c2R1ni5ks5qNcAfgaJpZM4I2N9T>
> .
>